.github/workflows/workflow.yml

```diff
@@ -31,6 +31,17 @@ jobs:
     strategy:
       matrix:
         ruby: ['2.5', '2.7']
```

Comment:
It probably wouldn't hurt to add `3.1`/`3.2` here as well since those are preferred for `Rails 7`. Anything to get Rails 7 support first though.

Reply:
I think it makes sense to add support for 3.0 and 3.1 as a separate PR? I'm happy to do that as soon as this is merged. We have used both 3.0 and 3.1 with hair_trigger so that should be no problem

Reply:
That makes sense to me, especially if there's a chance adding those to this PR delays getting it merged in.

Reply:
@erikaxel could you create a PR for rubies 3.0 and 3.1?